backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/domain/Card.kt

```diff
@@ -0,0 +1,28 @@
+package app.ehrenamtskarte.backend.verification.domain
+
+import java.time.LocalDateTime
+
+data class Card(val totpSecret: ByteArray, val expirationDate: LocalDateTime, val hashModel: String) {
+    val hasExpired: Boolean
+    get() = expirationDate.isAfter(LocalDateTime.now())
+
+    override fun equals(other: Any?): Boolean {
```

Comment:
Very sad that one can not use default equals here :(

However, Intelij says that contentEquals is marked as deprecated.

Reply:
Hm, IntelliJ does not complaint about it to me, but indeed it is marked as deprecated. But did you find out what to use else? The docs are not really useful there

Reply:
No, a quick search wasn't helpful for me, too

Reply:
Deprecated: Use Kotlin compiler 1.4 to avoid deprecation warning.

What a stupid suggestion in the kotlin docu. If you like a replacement for the outdated stuff do not use the version where it is oudated :D

Reply:
I changed it to `List<Byte>` as suggested by @Schedulaar, that makes all the `equals`/`hashCode` redundant